startupscript/aws/resource-mount.sh

```diff
@@ -25,5 +26,9 @@ else
 fi
 
 if [[ "${LOG_IN}" == "true" ]]; then
-  ${RUN_AS_LOGIN_USER} "wb resource mount"
+  source "${CLOUD_SCRIPT_DIR}/configure-aws-vault.sh"
+  ${RUN_AS_LOGIN_USER} "export AWS_VAULT_BACKEND='file' && \
```

Comment:
source ~/.bashrc?

Reply:
`source` doesn't work inside `sudo`, rolled this back.